### PR TITLE
Update DEPLOYMENT.md

### PR DESCRIPTION
**Pull request by ckprice**
We have a new QA team! Adding their steps to docs.

### CODE REVIEW

**Comment by ckprice** [comment]
@clouserw - need some clarification on the `Verify Production` section.

File: docs/DEPLOYMENT.md [hunk 1]
```diff
@@ -8,34 +8,59 @@ Note: we auto deploy the master branch to our *development environment*: [http:/
 
 ## Tag Release ##
 
-On the Monday after the milestone closes, Tag the release.
+This will happen on Friday at the end of sprint.
```
Comment (jaredhirsch) [inline docs/DEPLOYMENT.md:23]: 'This' is a bit vague. Maybe 'Releases should normally happen on Friday...' ?

File: docs/DEPLOYMENT.md [hunk 2]
```diff
@@ -8,34 +8,59 @@ Note: we auto deploy the master branch to our *development environment*: [http:/
...
 
 1. https://github.com/mozilla/testpilot/releases/new
 2. Tag Version: YYYY-MM-DD (append -N if more than one release is tagged on a given day: 2016-04-08-1)
 3. Release Title: YYYY-MM-DD
 4. Click `Publish`
 5. Wait for CircleCI to confirm that the release has passed ([tags uploaded to container](https://hub.docker.com/r/mozilla/testpilot/tags/)).
 
+Please be as detailed as possible in the release notes.
```
Comment (jaredhirsch) [inline docs/DEPLOYMENT.md]: It's probably good to either point to an example or list the details that should be included. The more explicit the docs are, the easier it'll be for people to take vacations and hand off chores ^_^

File: docs/DEPLOYMENT.md (hunk 1 above)
Comment (jaredhirsch) [inline docs/DEPLOYMENT.md:23]: Never mind, I see that sentence has been sprinkled throughout, it's fine

File: docs/DEPLOYMENT.md [hunk 3]
```diff
@@ -8,34 +8,59 @@ Note: we auto deploy the master branch to our *development environment*: [http:/
...
 
+This will happen on Friday at the end of sprint.
+
 Once we've confirmed our release passed CircleCI, and we've confirmed that it is uploaded into the container, we need to request for Ops to deploy to stage.
 
 This is done via a deployment bug.
 
 Example: https://bugzilla.mozilla.org/show_bug.cgi?id=1263682
 
+Include Softvision in the CC:
```
Comment (jaredhirsch) [inline docs/DEPLOYMENT.md:43]: What bugzilla accounts specifically should be included? The example doesn't CC anybody outside the dev team.

File: docs/DEPLOYMENT.md [hunk 4]
```diff
@@ -8,34 +8,59 @@ Note: we auto deploy the master branch to our *development environment*: [http:/
...
 
+This will happen on Friday at the end of sprint.
+
 Ops will inform us once we are deployed to our *staging environment*: [http://testpilot.stage.mozaws.net](http://testpilot.stage.mozaws.net).
 
 ## Test Stage ##
 
+This will happen on Friday at the end of sprint.
+
 Send out an email notification to `testpilot-dev@mozilla.com` to please test the staging environment.
```
Comment (jaredhirsch) [inline docs/DEPLOYMENT.md]: Is there a good example email that we could link to? Archives are here: https://mail.mozilla.org/pipermail/testpilot-dev/

File: docs/DEPLOYMENT.md [hunk 5]
```diff
@@ -8,34 +8,59 @@ Note: we auto deploy the master branch to our *development environment*: [http:/
...
 
-We have a [list of steps for basic functionality](https://docs.google.com/document/d/1cYelgp7dPwHD1eQIs4POgFqF2NOzxmVpKP4rnPR-RDA/edit), but new features should also be tested.
+Include Softvision and the bug link in the email notification.
```
Comment (jaredhirsch) [inline docs/DEPLOYMENT.md:57]: Again, I don't know what 'include softvision' means. Are there specific email addresses?

File: docs/DEPLOYMENT.md [hunk 6]
```diff
@@ -8,34 +8,59 @@ Note: we auto deploy the master branch to our *development environment*: [http:/
...
 ## Approve Production ##
 
-Once we are comfortable that the site has been tested, update the bug with approval to move to our *production environment*: [http://testpilot.firefox.com](http://testpilot.firefox.com).
\ No newline at end of file
+Once we are comfortable that the site has been tested, update the bug with approval to move to our *production environment*: [http://testpilot.firefox.com](http://testpilot.firefox.com).
+
+## Verify Production ##
+
+Softvision will verify production for us, and report any bugs.
+
```
Comment (jaredhirsch) [inline docs/DEPLOYMENT.md:88]: This might be a good opportunity to document our rollback process?

**Comment by jaredhirsch** [comment]
@ckprice Added a few comments. Thanks for keeping the docs maintained :beers:

File: docs/DEPLOYMENT.md (hunk 6 above)
Comment (lmorchard) [inline docs/DEPLOYMENT.md:88]: Yeah, we don't really have one & we should come up with one. We've done a simple release rollback at least once and got lucky. But we shouldn't reflexively reach for that option as a panic button when things break.

Rolling back to the previous release is dangerous / impossible when we have database migrations - meaning the site is likely to break on a rollback and require careful manual ops intervention to repair. In other words, compound the problem and require hours to get back to working order.

Ideally, we should "roll forward" with `git revert` to undo problematic commits and push a new release. Of course, that's also problematic if we don't know what commits caused a problem. Or if the problematic commits also included the database migrations - in which case, we also need to pair the `git revert` with a commit that undoes the migration.

File: docs/DEPLOYMENT.md (hunk 3 above)
Comment (ckprice) [inline docs/DEPLOYMENT.md:43]: For now, the list is Paul and Cosmin. I think we'll just keep it as Softvision team as we may not have the same team all time?

/edit: added a Softvision section at the top.

**Comment by ckprice** [comment]
Any other comments? I'll probably just merge this in a bit.

**Pull request merged**